Dockerfile-3

```diff
@@ -0,0 +1,47 @@
+FROM python:3-alpine3.6 

```

Comment:
pretty much same comments as for the above dockerfile

Reply:
Yes, all changes backported to this file as well